scripts/oddities/OddityProvider.js

```diff
@@ -3,7 +3,9 @@ let oddities = []
 
 
 export const useOddities = () => {
-    oddities.pop()
+    oddities = oddities.filter (oddity => {
+        return oddity.id < 52
```

Comment:
I suggest using hasOwnProperty() to make sure the data has the keys we need.

https://dmitripavlutin.com/check-if-object-has-property-javascript/